engine/time/src/main/java/io/deephaven/time/calendar/BusinessCalendar.java

```diff
@@ -1287,7 +1291,7 @@ public LocalDate[] businessDates(final LocalDate start, final LocalDate end) {
      * @throws InvalidDateException if the dates are not in the valid range
      * @throws DateTimeUtils.DateTimeParseException if the string cannot be parsed
      */
-    public LocalDate[] businessDates(final String start, final String end) {
+    public String[] businessDates(final String start, final String end) {
```

Comment:
Every other `businessDate` overload returns a `LocalDate[]`. I don't think the one that takes string inputs needs to be special.

Reply:
I assumed that String in implies String out.  If someone is working in strings, do you expect that they would not want this result in strings as well?